stack.yaml.lock

```diff
@@ -5,40 +5,54 @@
 
 packages:
 - completed:
-    hackage: hasql-dynamic-statements-0.3.1@sha256:c3a2c89c4a8b3711368dbd33f0ccfe46a493faa7efc2c85d3e354c56a01dfc48,2673
     pantry-tree:
       sha256: b1b9a6a26ec765e5fe29f9a670a5c9ec7067ea00dee8491f0819284ff0201b6f
       size: 641
+    hackage: hasql-dynamic-statements-0.3.1@sha256:c3a2c89c4a8b3711368dbd33f0ccfe46a493faa7efc2c85d3e354c56a01dfc48,2673
```

Comment:
Don't know why Stack modifies the order of the `hackage` line in my environment. Is this normal or should I change the version (I'm using  Stack ` 2.7.5`).

Reply:
Yeah I got those diffs too, they're a bit annoying, but I wouldn't worry about them.